utils/utils.go

```diff
@@ -236,3 +237,22 @@ func ForEachZone(zones []string, f func(zone string) error) error {
 
 	return meg.Wait().ErrorOrNil()
 }
+
+// ParseInstanceType returns an v3.InstanceType with family and name.
+func ParseInstanceType(instanceType string) v3.InstanceType {
```

Comment:
It would be nice to have unit test for this function, it's easy to write.

Reply:
https://github.com/exoscale/cli/pull/620/commits/eeccc70689056d0b68d4414ab845a72e91b91af4